recipes/delete-me/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/matteocao/{{ name }}.git
```

Comment:
Could you please get the source from tarballs instead of ``git_url``?
Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=tarballs#build-from-tarballs-not-repos